dietpi/func/dietpi-set_hardware

```diff
@@ -25,7 +25,7 @@ $FP_SCRIPT	soundcard		target_card (non-matching name for reset to default). Add
 $FP_SCRIPT	remoteir		odroid_remote/justboom_ir_remote # Odroid/RPi only
 $FP_SCRIPT	lcdpanel		target_panel (none to remove all)
 $FP_SCRIPT	headless		enable/disable
-$FP_SCRIPT	gpumemsplit		64/128/256 # RPi only
+$FP_SCRIPT	gpumemsplit		64/128/256... Range: 16-944, RPi only
```

Comment:
is his missing the `#`  before `RPi only`

Reply:
Since this is an informational string only (nothing is executed) it doesn't matter here. Probably I'll rework the whole thing by times to align with other help/usage print standards, like here: https://github.com/MichaIng/DietPi/blob/master/dietpi/dietpi-services#L15-L31

Reply:
ok clear